### ISSUE TITLE
E2E: e2e project does not work with vite-plugin

### ISSUE THREAD

**Issue by VincentThomas06** [LinbuduLab/nx-plugins#78]
Hi! I'm using the vite plugin with my react app. I think the problem is that the e2e tests doesn't start until the vite serve command finishes, but that command never finishes

```jsonc

// e2e project.json
"e2e": {
  "executor": "@nrwl/cypress:cypress",
  "options": {
    "cypressConfig": "apps/e2e/[name]/cypress.json",
    "devServerTarget": "[name]:serve",
    "headed": true
  },
  "configurations": {
    "production": {
      "devServerTarget": "[name]:serve:production"
    }
  }
}
```

```jsonc
// app project.json
"serve": {
  "executor": "nx-plugin-vite:serve",
  "options": {
    "configFile": "apps/[name]/vite.config.ts",
    "port": 3000,
    "host": true,
    "https": false,
    "root": "apps/[name]"
  }
}
```

Error:
```bash
[ vincent@PC ] :: 21:27:24 >> Codebase.js λ nx e2e thunder-e2e

> nx run thunder-e2e:e2e

i Nx-Vite [Start] Starting 

√ Vite server ready at http://localhost:3000

# nothing happens here... No test results or anything
```

**Comment by linbudu599** [comment]
I've never used nx cypress plugin before, so I need to try it once before I can locate where the problem is.

**Comment by VincentThomas06** [comment]
Did you see what the problem was?

**Comment by linbudu599** [comment]
> Did you see what the problem was?

I have confirmed the problem, but it requires a complete rewrite of this plugin, I will finish and release the latest version by the end of this week, really sorry for the inconvenience.

**Comment by linbudu599** [comment]
@VincentThomas06 This was fixed in the latest versiob `1.5.1`, you can have a try for Cypress integration now.

**Comment by VincentThomas06** [comment]
> @VincentThomas06 This was fixed in the latest versiob `1.5.1`, you can have a try for Cypress integration now.

Great! I'll see if it works and if it does, i'll close this issue.
